tooling/maven/camel-maven-plugin/src/main/java/org/apache/camel/maven/RouteCoverageMojo.java

```diff
@@ -192,12 +205,37 @@ public void execute() throws MojoExecutionException, MojoFailureException {
 
         List<CamelNodeDetails> routeIdTrees = routeTrees.stream().filter(t -> t.getRouteId() != null).collect(Collectors.toList());
         List<CamelNodeDetails> anonymousRouteTrees = routeTrees.stream().filter(t -> t.getRouteId() == null).collect(Collectors.toList());
-
+                
+     // creates the folder for the xml.file
+        
+        File file =  (new File(project.getBasedir() + "/target/site/jacoco"));
+        if (file.exists()){
```

Comment:
We dont need this info logging as its just annoying. Instead it can be something about "generating jacoco report".